service/src/test/java/uk/gov/hmcts/reform/pebble/AgeFilterTest.java

```diff
@@ -12,7 +12,7 @@
 /**
  * Filter calculates age based on provided date and returns it in string format.
  *
- * Note: At the moment only supported input format is ISO formatted string date.
+ * <p>Note: At the moment only supported input format is ISO formatted string date.
```

Comment:
Typo here Damian (correct me if i'm wrong)

Reply:
Nope, it is JavaDoc syntax.